### ISSUE TITLE
1.14.12 DI - Rotate by 90deg does not work on heatmap window

### ISSUE THREAD

**Issue by rmadhura-iit** [biocatiit/musclex#52]
The `Rotate 90 degrees` should change alignment by 90 degrees. It is working on `Elliptical map` but it doesn't work on `Arrow Orientation`. 

Also the when the `Orientation Display` check box is checked, we only see circles in all the pixels. Not sure if this is a feature or an issue. Please clarify. 

**Before selecting `Rotate 90 degrees`**
![image](https://user-images.githubusercontent.com/43549103/62565849-51731580-b84d-11e9-9278-c1408b933c00.png)

**After selecting `Rotate 90 degrees`**
![image](https://user-images.githubusercontent.com/43549103/62565907-6d76b700-b84d-11e9-812b-b9fba82a2ea2.png)

**`Orientation Display` Issue**
![image](https://user-images.githubusercontent.com/43549103/62565958-867f6800-b84d-11e9-964e-84cbdc02441e.png)

**Comment by grantnikseresht** [comment]
The orientation display default is to show the ellipses that were present in previous versions when orientation display was checked, so this is a feature. It seems like we may want to switch to arrows by default, as it looks like the ellipses are difficult to interpret with the fixed aspect ratio.

**Comment by rmadhura-iit** [comment]
I agree, switching to arrows as default is a better idea. If we want to look at ellipses, we can chose the Elliptical map option anyway.

**Comment by rmadhura-iit** [comment]
We have to fix the rotate by `90deg` option in the next version though. This is important, particularly for brain maps, where the alignment of the species of interest may be 90deg from the diffraction alignment. For instance, myelin around neurons has lamellar packing structure which gives a diffraction pattern that is at right angle to the axial direction.
